doc/classes/AcceptDialog.xml

```diff
@@ -21,8 +21,8 @@
 			<argument index="2" name="action" type="String" default="&quot;&quot;">
 			</argument>
 			<description>
-				Add custom button to the dialog and return the created button.
-				The button titled with [i]text[/i] and the [i]action[/i] will be passed to [custom_action] signal when it is pressed.
+				Adds a button with label [i]text[/i] and a custom [i]action[/i] to the dialog and returns the created button. [i]action[/i] will be passed to the [custom_action] signal when pressed.
+				If `true`, [i]right[/i] will place the button to the right of any sibling buttons. Default value: `false`.
```

Comment:
`[code]true[/code]` instead of `` `true` ``?

Reply:
Cool, can change. The boolean example here used backticks so I wasn't quite sure: https://github.com/GDquest/godot-3-guides/blob/master/contribution-guides/class-reference-styleguide.md

Reply:
I'll ask GDquest about that.